kserve/README.md

```diff
@@ -0,0 +1,121 @@
+# KServe
+
+KServe comes with two component:
+
+1. [KServe](#KServe)
+2. [KServe runtimes](#KServe-Runtimes)
+
+## KServe
+
+Contains deployment manifests for the KServe controller.
+
+- [kserve-controller](https://github.com/opendatahub-io/kserve)
+  - Forked upstream kserve/kserve repository
+
+## KServe runtimes
+
+Contains the runtime manifests for KServe.
+
+- [kserve-controller](https://github.com/opendatahub-io/kserve)
+  - Forked upstream kserve/kserve repository
+
+## Original manifests
+
+KServe also uses `kustomize` so we can directly use [their manifests](https://github.com/kserve/kserve/tree/master/config).
+
+* `default` is the entrypoint for CRDs, KServe controller and RBAC resources.
+* `runtimes` is the second entrypoint for the KServe runtimes. They are referenced separately, as these are not namespaced.
+
+### Updating the KServe manifests
+
+Use the script in `hack` to update the KServe manifests:
+
+```bash
+./hack/update-kserve-manifests.sh
+```
+
+## Overlays
+
+There are two overlays defined with the necessary changes for ODH:
+
+* [controller](./odh-overlays/controller)
+* [runtimes](./odh-overlays/runtimes)
+
+
+### Installation process
+
+Following are the steps to install Model Mesh as a part of OpenDataHub install:
+
+1. Install the OpenDataHub operator
+2. Make sure you install Service Mesh and Serverless components and configure them appropriately
+3. Create a KfDef that includes the KServe components and runtimes
```

Comment:
have you tested this kfdef with your change?

Reply:
No, I just tested it with the kustomize part, as I do not know how to use it/what the full process is. Also there was the open question about the multiple entrypoints.

Reply:
Hm what about going back to having a script that gets the resources from KServe repo, then running `kustomize build` using our overlays and use the generated output as entrypoint for kfctl? Not super clean, but would solve all issues around kustomize without having to modify the KServe resources.
Other ideas? @Jooho, @israel-hdez

Reply:
> running `kustomize build` using our overlays and use the generated output as entrypoint for kfctl?

This would be quite fine IMO. Although, I still think we don't need to copy the files. I mean, I think running `kustomize build` will still generate what we need without copying files. Then, I guess the only thing left is to add operator parameters, but this should be a quite small task.

Reply:
This script will be used for opendatahub and rhods. So we need to images and some parts dynamically. If we generate final manifest by `kustomize build`, we can not reuse this manifests.

Reply:
@israel-hdez Exactly. I'll update the PR to showcase what I imagined.

Reply:
@ReToCode  It is looking good to me. Let's wait for @Jooho thoughts.

Reply:
@LaVLaS @VaishnaviHire what do you think about this approach for RHODS/ODH?

Reply:
@LaVLaS @VaishnaviHire gentle ping.